pkg/scheduler/util/utils.go

```diff
@@ -135,8 +135,14 @@ func (l *SortableList) Sort() {
 }
 
 // HigherPriorityPod return true when priority of the first pod is higher than
-// the second one. It takes arguments of the type "interface{}" to be used with
-// SortableList, but expects those arguments to be *v1.Pod.
+// the second one. If two pods' priorities are equal, compare their StartTime.
+// It takes arguments of the type "interface{}" to be used with SortableList,
+// but expects those arguments to be *v1.Pod.
```

Comment:
This change is fine, but we should rename this function to something like `MoreImportantPod` as it checks more than just priority.

Reply:
Addressed.